openfin-csharp-api-test/openfin-csharp-api-test/OpenfinTests.cs

```diff
@@ -181,8 +183,10 @@ public async Task AppEventsInitiallyOpen()
             };
 
             StopOpenfinApp();
+            await Task.Delay(500);
```

Comment:
I believe only this one was required to get the the CI tests to pass.
But I thought for the sake of avoiding future timing issues, adding an extra 1.5s to the build was acceptable